django_components/component.py

```diff
@@ -59,7 +62,19 @@ def __new__(mcs, name, bases, attrs):
         return super().__new__(mcs, name, bases, attrs)
 
 
-class Component(metaclass=SimplifiedInterfaceMediaDefiningClass):
+class ComponentView(View):
+    http_method_names = [
+        "get",
+        "post",
+        "put",
+        "patch",
+        "delete",
+    ]
```

Comment:
I don't think this is needed really. My comment about sending an OPTIONS request is as valid with a PUT or PATCH request: "Why would you want...?". But if we are making it a view, lets go all the way. This just inheriting from View is fine here.